src/features/document/document_feature.py

```diff
@@ -57,24 +57,20 @@ def get_by_path(
     return get_document_by_path_use_case(user=user, absolute_path=absolute_path)
 
 
-@router.put("/{data_source_id}/{document_id}", operation_id="document_update", responses=responses)
+@router.put("/{id_reference:path}", operation_id="document_update", responses=responses)
 @create_response(JSONResponse)
 def update(
-    data_source_id: str,
-    document_id: str,
+    id_reference: str,
     data: Json = Form(...),
-    attribute: Optional[str] = Form(None),
     files: Optional[List[UploadFile]] = File(None),
     update_uncontained: Optional[bool] = False,
     user: User = Depends(auth_w_jwt_or_pat),
 ):
     """Update document"""
```

Comment:
```suggestion
    """Update document
        - **id_reference**: <data_source>/<document_uuid> (can also include an optional .<attribute> after <document_uuid>)
     """
```

to add more documentation to swagger ui